// const err = new Error("Error arrived");
// console.log(err.stack);

const {CustomError} = require('./CustomEror');
throw new CustomError('Error arrived');








